src/sql/connection/connection.py

```diff
@@ -727,7 +727,7 @@ def _connection_execute(self, query, parameters=None):
         # TODO: we can parse the query to ensure that it's a SELECT statement
         # for example, it might start with WITH but the final statement might
         # not be a SELECT
-        is_select = first_word_statement in {"select", "with", "from"}
+        is_select = first_word_statement in {"select", "with", "from", "summarize"}
```

Comment:
please add a note explaining why we're adding "summarize" here

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/841/files#diff-62e3d6289e3242b271e6df3a7f201ab003f8ce73a5daf1a824cc2b4d892513d1R735)